docs/v0.7/en/reference/sql/information-schema/greptime-region-peers.md

```diff
@@ -0,0 +1,39 @@
+# GREPTIME_REGION_PEERS
+
+The `GREPTIME_REGION_PEERS` table shows detailed information of a single Region node in GreptimeDB, such as whether it is a learner or leader.
+
+:::tip NOTE
+This table is not available on [GreptimeCloud](https://greptime.cloud/).
```

Comment:
@killme2008 please confirm it

Reply:
confirm what? It's not available on cloud for security reason.